bruker2nifti/cli/bruker2nii.py

```diff
@@ -1,8 +1,11 @@
-import os
-import sys
 import argparse
+from datetime import datetime, timedelta
+import re
+import sys
+
 from bruker2nifti.converter import Bruker2Nifti
 import bruker2nifti._utils as utils
+from ..metadata import BrukerMetadata
```

Comment:
Can you use the absolute path, within the packaging best practice? (`bruker2nifti._metadata`, and change the package name from `metadata` to `_metadata`? For some reason I wanted to expose only the interface.)

Reply:
... I see the import patterns had not been consistent across the code, e.g.
https://github.com/SebastianoF/bruker2nifti/blob/master/bruker2nifti/converter.py#L3
If you have time, please feel free to bring it all to the same standard with the absolute path!

Reply:
Updated across all files now I think.